include/rest-request.php

```diff
@@ -39,14 +61,14 @@ public function init() {
 			$this->filters_links = new PLL_Filters_Links( $this );
 			$this->filters = new PLL_Filters( $this );
 
-			// Static front page and page for posts
+			// Static front page and page for posts?
```

Comment:
Comments should provide answers to the readers. They shouldn't raise more questions.